### PR TITLE
Update URL for Debian cloud images

### PR DESCRIPTION
**Pull request by maximiliankolb**
Debian 10 "Buster" is EoL but the Debian project offers multiple cloud images for multiple cloud providers similar to Canonical.

* [x] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.11/Katello 4.13

### CODE REVIEW

File: guides/common/modules/proc_creating-an-image-for-amazon-ec2.adoc [hunk 1]
```diff
@@ -14,7 +14,7 @@ You can create images for Amazon EC2 from within {Project}.
 * In the *Username* field, enter the username needed to SSH into the machine.
 * In the *Image ID* field, enter the image ID provided by Amazon or an operating system vendor.
 ifndef::satellite[]
-This ID can be found within Amazon AWS or on operating system specific pages like https://wiki.debian.org/Cloud/AmazonEC2Image/Buster[debian.org] or https://cloud-images.ubuntu.com/locator/ec2/[ubuntu.com].
+This ID can be found within Amazon AWS or on operating system specific pages like https://cloud.debian.org/images/cloud/[debian.org] or https://cloud-images.ubuntu.com/locator/ec2/[ubuntu.com].
```
Comment (apinnick) [inline guides/common/modules/proc_creating-an-image-for-amazon-ec2.adoc:17]:
```suggestion
You can find the ID in Amazon AWS or on operating system specific pages such as https://cloud.debian.org/images/cloud/[debian.org] or https://cloud-images.ubuntu.com/locator/ec2/[ubuntu.com].
```
Please change passive voice to active. 
The second suggestion  ("like" to "such as") is purely subjective, not required.

**Review by apinnick — CHANGES_REQUESTED** [review]

File: guides/common/modules/proc_creating-an-image-for-amazon-ec2.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_creating-an-image-for-amazon-ec2.adoc]: Applied both suggestions.

**Review by maximiliankolb — COMMENTED** [review]
Ready for re-review.

**Review by apinnick — APPROVED** [review]
LGTM

**Comment by apinnick** [comment]
Merged to 'master' and cherry-picked:

f707d9cbdd..c59386bec4  3.11 -> 3.11

**Pull request merged**